lib/capybara/driver/node.rb

```diff
@@ -138,6 +156,27 @@ def inspect
       def ==(other)
         eql?(other) || (other.respond_to?(:native) && native == other.native)
       end
+
+      protected
+
+      def normalize_spacing(text)
+        text
+          .delete(REMOVED_CHARACTERS)
+          .tr(SQUEEZED_SPACES, ' ')
+          .squeeze(' ')
+          .sub(LEADING_SPACES, '')
+          .sub(TRAILING_SPACES, '')
```

Comment:
These ones are `\A` and `\z` beginning and end bound, so there can only really be one instance of them in a string. Probably a _slightly_ faster way to do this